docs/storage/custom-storage-domains.md

```diff
@@ -0,0 +1,62 @@
+﻿
+
+# Custom Storage API Domains
+
+Upon creation of a Fleek.co Storage bucket, users are assigned a default API URL that looks something like: 'Storageapi.fleek.one` + bucket ID.
+
+With Custom Storage API Domains, we're increasing the uptime, availability, and accessibility of these storage buckets by allowing existing Fleek.co storage users to add their own custom domains to map to their Fleek storage buckets.
+
+!!! info
+
+    When you deploy on Fleek your site will have an IPFS hash and future automatic deployments will also update the IPFS hash of your site. Your site comes with a CDN, free SSL certificate, and a Fleek preview URL. You can add Custom Domains, ENS Domains, Configure Deploy Previews, and more.
+
+## Add Custom Domain
+
+Navigate to the storage tab, denoted by a `new` prompt and found in the vertical navigation bar of the Fleek.co app.
+
+![](imgs/add-domain1.png)
+
+#### 1.  Click `Add a Custom Domain`
+
+#### 2.  Enter the Domain you want to map to your storage bucket -- make sure this is a domain you own.

```

Comment:
I think we are missing the screenshot for this, i.e, with the modal to enter the domain open

Reply:
It's added now but in the wrong place. The first image should come after step 1, and this new image should come after step 2 right? Not sure but the ordering looks weird and the first step is missing the number.